nixos/modules/services/x11/hardware/libinput.nix

```diff
@@ -198,6 +198,13 @@ in {
 
     environment.systemPackages = [ pkgs.xorg.xf86inputlibinput ];
 
+    environment.etc = [
+      (let cfgPath = "/X11/xorg.conf.d/40-libinput.conf"; in {
+        source = pkgs.xorg.xf86inputlibinput.out + "/share" + cfgPath;
+        target = cfgPath;
```

Comment:
This works, but `target` is not expected to start with `/`.

Reply:
Thanks, fixed.